contents/handbook/company/goal-setting.md

```diff
@@ -6,31 +6,39 @@ showTitle: true
 
 ## How it works
 
-Every [small team](../people/team-structure/why-small-teams) has:
-
-- As few Objectives as possible - ideally 1
-- Motivation - explains why the objective is set
-- Key Results that show if we're en route to achieving an Objective
+1. Every three months, a couple of weeks before the end of the period, James and Tim (+ other execs) come up with the company goals.
+2. They will then propose goals for every [small team](../people/team-structure/why-small-teams) by creating PRs against the small team pages.
+3. Each small team can change or propose alternate goals
+4. After the end of the period we'll run an all hands retrospective
 
 ## Why we do this
 
 - Sense of winning / losing - this bonds teams, increases urgency and creates satisfaction when we achieve things
 - Clarifies our direction
 
-## When we do this
+## Good goal setting
 
-- We _do not_ do this for every team at the same time - if it takes a forced process to keep these up-to-date, and so people look at them, then the objectives aren't meaningful
-- It's up to the small team leaders to set a cadence for their own objectives and key results. Changes need to be communicated to James (who makes sure they stack up with each other properly)
-- Objectives should have a deadline
+Use this template:
 
-## Good goal setting
+```md
+## March/June/September/December 202x goals
 
+1. PostHog in the EU
+Key results
+  - 100 customers using EU cloud
+  - 99% uptime
+  - Time spent maintaining < 1 hour/week
+  - Uses the same helm chart as US cloud
+```  
+
+- As few Objectives as possible - ideally 1
+- Motivation - explains why the objective is set
```

Comment:
Maybe add motivation to the above example too

Reply:
Good feedback, done.